TransferMethod/Sources/SelectTransferMethodTypePresenter.swift

```diff
@@ -290,4 +303,30 @@ final class SelectTransferMethodTypePresenter {
     func setInsights(hyperwalletInsights: HyperwalletInsightsProtocol) {
         self.hyperwalletInsights = hyperwalletInsights
     }
+
+     private func trackTransferMethodClick() {
+        let impressionParams = [
+            InsightsTags.country:
+                self.selectedCountry,
+            InsightsTags.currency:
+                self.selectedCurrency,
+            InsightsTags.transferMethodType:
+                self.selectedTransferMethodType
+        ]
+      getInsights().trackImpression(pageName: pageName, pageGroup: pageGroup, params: impressionParams)
+  }
+    private func trackCountryClick() {
+           getInsights()
+               .trackClick(pageName: pageName,
+                           pageGroup: pageGroup,
+                           link: link,
+                           params: [InsightsTags.country: self.selectedCountry])
+       }
+    private func trackCurrencyClick() {
+          getInsights()
+          .trackClick(pageName: pageName,
+                     pageGroup: pageGroup,
+                     link: link,
```

Comment:
same as above, look for link value for Select Currency usecase

Reply:
Changed the link value